recipes/_master_base_config.rb

```diff
@@ -28,6 +28,38 @@
 # Get VPC CIDR
 node.default['cfncluster']['ec2-metadata']['vpc-ipv4-cidr-block'] = get_vpc_ipv4_cidr_block(node['macaddress'])
 
+# Get shared_dir path and mount EFS filesystem
+efs_shared_dir = "/" + node['cfncluster']['cfn_efs_shared_dir'].split(',')[0]
+
+# Check to see if EFS is created
+if efs_shared_dir != "/NONE"
+  # Create the EFS shared directory
+  directory efs_shared_dir do
+    owner 'root'
+    group 'root'
+    mode '1777'
+    recursive true
+    action :create
+  end
+
+  # Mount EFS over NFS
+  mount efs_shared_dir do
+    device "#{node['cfncluster']['cfn_efs']}.efs.#{node['cfncluster']['cfn_region']}.amazonaws.com:/"
+    fstype 'nfs4'
+    options 'nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=30,retrans=2,noresvport,_netdev'
+    dump 0
+    pass 0
+    action %i[mount enable]
+  end
+
+  # Make sure EFS shared directory permissions are correct
+  directory efs_shared_dir do
```

Comment:
Do we really need it? It is the same code executed in the first step.

Reply:
Isn't this needed on the compute node, or this operation performed on the master is enough?

Reply:
This action is not needed on the compute for some reason. However since I am moving efs mounting to a separate recipe and including the recipe in the master and compute base config, we might as well just do this for the compute as well for extra assurance